src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -110,23 +125,26 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
             />
           )}
 
-          <TravellerSelection
-            setTravellerSelection={setTravellerSelection}
-            preassignedFareProduct={preassignedFareProduct}
-            selectableUserProfiles={selectableTravellers}
-            style={styles.selectionComponent}
-          />
+          {travellerSelectionMode !== 'none' && (
```

Comment:
I think this should be encapsulated inside `TravellerSelection`, and the same for `Zones` and `StartTimeSelection`. This makes the `Overview`-component less complex as it don't need to know how the selection-modes work, that is for the sub-component to handle.